packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -635,6 +636,16 @@ class PlatformViewLayer extends Layer {
   /// A UIView with this identifier must have been created by [PlatformViewsServices.initUiKitView].
   final int viewId;
 
+  /// [MouseTrackerAnnotation] associated with the platform view layer.
+  ///
+  /// Gesture recognizers don't receive hover events due to the performance
+  /// cost associated with hit testing a sequence of potentially thousands of
+  /// events -- move events only hit-test the down event, then cache the result
+  /// and apply it to all subsequent move events, but there is no down event
+  /// for a hover. To support native hover gesture handling by platform views,
+  /// we attach/detach this layer annotation as necessary.
```

Comment:
I suggest describe more about what this parameter does:

```dart
/// Assign this parameter to receive mouse events on the region of this platform
/// view.
///
/// If [hoverAnnotation] is not null, [PlatformViewLayer] will annotate the
/// region of this platform view with this annotation, so that the callbacks of
/// this annotation can receive mouse events, which includes mouse enter, exit,
/// and hover, but does not include mouse down, move, and up. The layer will
/// also appear as opaque during an annotation search, roughly meaning it will
/// prevent layers behind it from receiving the events.
///
/// By default, [hoverAnnotation] is empty, and [PlatformViewLayer] will not
/// receive mouse events, and will appear translucent during the annotation
/// search.
///
/// See also:
///
///  * [MouseRegion], which explains more about the mouse events and opacity
///    during annotation search.
```

I removed the reason for not receiving mouse events, but feel free to add it if you feel like.

Reply:
Awesome -- Done with minor tweaks to keep it style-compliant.